clients/java/src/main/java/io/camunda/zeebe/client/impl/worker/JobWorkerBuilderImpl.java

```diff
@@ -151,27 +136,14 @@ public JobWorkerBuilderStep3 backoffSupplier(final BackoffSupplier backoffSuppli
   public JobWorker open() {
     ensureNotNullNorEmpty("jobType", jobType);
     ensureNotNull("jobHandler", handler);
-    ensureGreaterThan("timeout", timeout, 0L);
+    ensureNotNull("timeout", timeout);
```

Comment:
Thank you, but can you take a look at https://github.com/camunda/zeebe/pull/11999/files#diff-81afc9878bd53bc462f13a3ec995e7560fceebfe9251d066a140fc1b2520c86dR46?

The `timeout` is a `Duration` type right now that could not be negative, as I understand it. WDYT?

Reply:
Please refer to [the reference documentation of Duration class](https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html) where it explains that it can be negative.

Reply:
> Please refer to [the reference documentation of Duration class](https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html) where it explains that it can be negative.

My assumption was embarrassing, thank you for the info 🙂